README.md

```diff
@@ -1564,6 +1567,110 @@ for domain in domains:
     print(domain)
 ```
 
+### Create Static brcode
+
+Dynamic Brcodes store account information in the form of a Pix Key and can easily be used to create Pix transactions.
+
+```python
+import starkinfra
+
+brcodes = starkinfra.staticbrcode.create(
+    brcodes=[
+        starkinfra.StaticBrcode(
+            name="Jamie Lannister",
+            key_id="+5511988887777",
+            amount=100,
+            reconciliation_id="123",
+        )
+    ]
+)
+
+for brcode in brcodes:
+    print(brcode)
+```
+
+### Query Static brcode
+
+You can query multiple Static brcodes according to filters.
```

Comment:
Usa sempre ou o nome comum (static BR Code) ou o nome do recurso (StaticBrcode):

You can query multiple StaticBrcodes according to filters.